packages/testsuite/cypress/e2e/undertow/test-configuration-subsystem-undertow-global-settings.cy.ts

```diff
@@ -0,0 +1,114 @@
+        managementEndpoint = result as string;
+      })
+      .then(() => {
+        // create fixtures
+        cy.executeInWildflyContainer(`/subsystem=undertow/server=${testValues.serverName}:add`);
+        cy.executeInWildflyContainer(`/subsystem=undertow/servlet-container=${testValues.servletContainer}:add`);
+
+        cy.executeInWildflyContainer(`/subsystem=undertow/server=new-server/host=${testValues.serverHost}:add`);
+      });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  beforeEach(() => {
+    cy.navigateTo(managementEndpoint, "undertow");
+    // the form takes a brief moment to initialize
+    cy.wait(200);
+  });
+
+  it("Should update the default security domain configuration ", () => {
+    cy.editForm(globalSettingsForm.formId);
+    cy.text(globalSettingsForm.formId, globalSettingsForm.defaultSecurityDomain, "some test value");
+    cy.saveForm(globalSettingsForm.formId);
+    cy.verifySuccess();
+    cy.verifyAttribute(managementEndpoint, address, "default-security-domain", "some test value");
+  });
+
+  it("Should update the default server configuration", () => {
+    cy.editForm(globalSettingsForm.formId);
+    cy.text(globalSettingsForm.formId, globalSettingsForm.defaultServer, testValues.serverName);
+    cy.saveForm(globalSettingsForm.formId);
+    cy.verifySuccess();
+    cy.verifyAttribute(managementEndpoint, address, "default-server", testValues.serverName);
+  });
+
+  it("Should update the default servlet container configuration", () => {
+    cy.editForm(globalSettingsForm.formId);
+    cy.text(globalSettingsForm.formId, globalSettingsForm.defaultServletContainer, testValues.servletContainer);
+    cy.saveForm(globalSettingsForm.formId);
+    cy.verifySuccess();
+    cy.verifyAttribute(managementEndpoint, address, "default-servlet-container", testValues.servletContainer);
+  });
+
+  it("Should update the default virtual host configuration", () => {
+    cy.editForm(globalSettingsForm.formId);
+    cy.text(globalSettingsForm.formId, globalSettingsForm.defaultVirtualHost, testValues.serverHost);
+    cy.saveForm(globalSettingsForm.formId);
+    cy.verifySuccess();
+    cy.verifyAttribute(managementEndpoint, address, "default-virtual-host", testValues.serverHost);
+  });
+
+  it("Should update the instance ID configuration", () => {
+    cy.editForm(globalSettingsForm.formId);
+    cy.text(globalSettingsForm.formId, globalSettingsForm.instanceId, testValues.instanceId);
+    cy.saveForm(globalSettingsForm.formId);
+    cy.verifySuccess();
+    cy.verifyAttribute(managementEndpoint, address, "instance-id", testValues.instanceId);
+  });
+
+  it("Should update the statistics-enabled configuration by changing the expression", () => {
+    console.log("forgodssake");
+    cy.editForm(globalSettingsForm.formId);
+    const expressionFormInput = "input#undertow-global-settings-form-statistics-enabled-editing.form-control";
+    cy.get(expressionFormInput).click().wait(200).clear();
```

Comment:
> `.wait(200)`

Why do we need to wait?

Reply:
the wait will do small pause between command. If you need increase time required for loading JS add second parameter to method

> get(expressionFormInput, {timeout=200}).click({timeout=200}).clear();

anyway @OndrejKotek prepare the PR where is increased default timeout. see: https://github.com/hal/berg/pull/100

Reply:
In fact it does not need an increased timeout, but a little wait time after clicking before clearing. 100ms has not proved sufficient in testing - 200ms seems to be the least I can get away with reliably.

Reply:
this was now refactored away (the `cy.text()` uses the same wait time)